docs/appendix/try-viam/reserve-a-rover.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Reserve a Viam Rover"
+linkTitle: "Reserve a Viam Rover"
+weight: 10
+type: "docs"
+description: "Reserve a Viam Rover located on-site at Viam in NYC."
+images: ["/appendix/try-viam/try-viam-reserve-preview.png"]
+imageAlt: "Rover reservation page"
+tags: ["try viam", "app"]
+aliases:
+  - "/try-viam/reserve-a-rover/"
+  - "/get-started/try-viam/reserve-a-rover/"
+---
+
+_Try Viam_ is a way to try out the Viam platform without setting up any hardware yourself.
+You can take over a Viam Rover in our robotics lab to play around!
+
+Watch this tutorial video for a walkthrough of Try Viam, including [how to reserve a Viam Rover](#using-the-reservation-system), [navigate the Viam platform](/fleet/), and [drive the rover](/components/base/wheeled/#test-the-base):
+
+{{<youtube embed_url="https://www.youtube-nocookie.com/embed/YYpZ9CVDwMU" max-width="600px">}}
+
+## Using the reservation system
+
+### Create a reservation
+
+{{< readfile "/static/include/try-viam/create-a-reservation.md" >}}
+
+### Access your rover rental
+
+Once your reservation starts and the system has configured your rover, click **TRY MY ROVER** from the [**TRY VIAM** page](https://app.viam.com/try) or, if you were queuing, click **Take Me to My Rover** in the confirmation email.
+
+{{<gif webm_src="/appendix/try-viam/rover-reservation.webm" mp4_src="/appendix/try-viam/rover-reservation.mp4" alt="Rover reservation management page" max-width="1000px">}}
+
+{{<imgproc src="appendix/try-viam/navigation-bar.png" resize="800x" alt="Navigation bar of the Viam app with the Viam Rover time remaining indicator.">}}
+
+### Limitations
+
+When using a rented Viam rover, adding [modules](/registry/) is disabled for security purposes.
+
+### Extend your reservation
+
+{{< readfile "/static/include/try-viam/extend-a-reservation.md" >}}
+
+### Cancel my reservation/session
+
+{{< readfile "/static/include/try-viam/cancel-a-reservation.md" >}}
+
+## Next steps
+
+{{< cards >}}
+{{% card link="/get-started/drive-rover/" %}}
+{{% card link="/tutorials/services/basic-color-detection/" %}}
+{{< /cards >}}
+
+## FAQ
+
+Try Viam allows you to try the Viam platform without setting up any hardware yourself.
+No matter where you are in the world you can take over and play around with a Viam Rover in our robotics lab!
+
+### How do I make a reservation to take over a Viam Rover?
+
+{{< readfile "/static/include/try-viam/create-a-reservation.md" >}}
+
+### My machine had an error, a system crash, or is physically stuck
+
+1. Please notify Viam support on [our Community Discord](https://discord.gg/viam).
+2. Use the **Add Viam Support** button on your machine's Location page to give Viam Support access to your _location_.
```

Comment:
I assume because if they come to discord we can help them with the rest. And they may assume we get a notification if they add support and I am not sure we do.